robot/Cumulus/tests/browser/donations/create_donation_payment.robot

```diff
@@ -1,36 +1,68 @@
 *** Settings ***
 
 Resource        robot/Cumulus/resources/NPSP.robot
-Suite Setup     Open Test Browser
+Library         cumulusci.robotframework.PageObjects
+...             robot/Cumulus/resources/ContactPageObject.py
+...             robot/Cumulus/resources/AccountPageObject.py
+...             robot/Cumulus/resources/OpportunityPageObject.py
+...             robot/Cumulus/resources/NPSP.py
+Suite Setup     Run keywords
+...             Open Test Browser
+...             Setup Test Data
 Suite Teardown  Delete Records and Close Browser
 
+***Keywords***
+# Sets up all the required data for the test based on the keyword requests
+Setup Test Data
+    Setupdata   contact   ${contact1_fields}
+
+*** Variables ***
+&{contact1_fields}       Email=test@example.com
+
 
 *** Test Cases ***
 
 Create Donation and Opportunity and Create Payment Manually
-    [tags]  unstable
-    &{contact} =  API Create Contact    Email=skristem@robot.com
-    Store Session Record    Account    &{contact}[AccountId]
-    Go To Object Home         Opportunity
-    Click Object Button       New
-    Select Record Type        Donation
-    Create Opportunities    Sravani $100 donation    &{Contact}[LastName] Household    Closed Won
-    Save Current Record ID For Deletion    Opportunity   
-    Select Tab    Related
-    Load Related List    Payments
-    Verify Occurrence    Payments    0
-    Click Related List Button    Payments    New
+    [Documentation]  Navigate to Opportunities page and open an Opportunity>In the right sections, go to Payments click on drop down Arrow>New
+    ...              Create a new payment for the Opportunity.
+
+    [tags]                                          W-038461                 feature:Donations
+
+    Go To Page                                      Listing
+    ...                                             Opportunity
+
+    Click Object Button                             New
+    Select Record Type                              Donation
+
+    Create Opportunities                            Sravani $100 donation
```

Comment:
Remove the create opportunities keyword from npsp.robot file and move everything to here as I am not sure if any other test case is using it. Also before making that change confirm that no other test case is using and if using make appropriate changes

Reply:
Create Opportunities is being used in other places such as grants. Removing it now may cause the fail. Will address when I am refactoring those tests.

Reply:
Done